from PyQt5 import QtCore, QtGui, QtWidgets

class Ui_HookCollectionView(object):
    def setupUi(self, Form):
        Form.setObjectName("Form")
        Form.resize(810, 293)
        self.gridLayout = QtWidgets.QGridLayout(Form)
        self.gridLayout.setObjectName("gridLayout")
        self.comboBox_21 = QtWidgets.QComboBox(Form)
        self.comboBox_21.setObjectName("comboBox_21")
        self.comboBox_21.addItem("")
        self.comboBox_21.addItem("")
        self.gridLayout.addWidget(self.comboBox_21, 5, 2, 1, 4)
        self.pushButton_46 = QtWidgets.QPushButton(Form)
        self.pushButton_46.setObjectName("pushButton_46")
        self.gridLayout.addWidget(self.pushButton_46, 4, 3, 1, 1)
        self.label_11 = QtWidgets.QLabel(Form)
        self.label_11.setAlignment(QtCore.Qt.AlignCenter)
        self.label_11.setObjectName("label_11")
        self.gridLayout.addWidget(self.label_11, 2, 2, 1, 2)
        self.label_12 = QtWidgets.QLabel(Form)
        self.label_12.setAlignment(QtCore.Qt.AlignCenter)
        self.label_12.setObjectName("label_12")
        self.gridLayout.addWidget(self.label_12, 7, 2, 1, 3)
        self.pushButton_45 = QtWidgets.QPushButton(Form)
        self.pushButton_45.setObjectName("pushButton_45")
        self.gridLayout.addWidget(self.pushButton_45, 4, 2, 1, 1)
        self.pushButton_47 = QtWidgets.QPushButton(Form)
        self.pushButton_47.setObjectName("pushButton_47")
        self.gridLayout.addWidget(self.pushButton_47, 4, 4, 1, 1)
        self.comboBox_22 = QtWidgets.QComboBox(Form)
        self.comboBox_22.setObjectName("comboBox_22")
        self.comboBox_22.addItem("")
        self.comboBox_22.addItem("")
        self.gridLayout.addWidget(self.comboBox_22, 6, 2, 1, 4)
        self.pushButton_48 = QtWidgets.QPushButton(Form)
        self.pushButton_48.setObjectName("pushButton_48")
        self.gridLayout.addWidget(self.pushButton_48, 4, 5, 1, 1)
        self.horizontalLayout_3 = QtWidgets.QHBoxLayout()
        self.horizontalLayout_3.setObjectName("horizontalLayout_3")
        self.pushButton = QtWidgets.QPushButton(Form)
        self.pushButton.setObjectName("pushButton")
        self.horizontalLayout_3.addWidget(self.pushButton)
        self.pushButton_3 = QtWidgets.QPushButton(Form)
        self.pushButton_3.setObjectName("pushButton_3")
        self.horizontalLayout_3.addWidget(self.pushButton_3)
        self.pushButton_2 = QtWidgets.QPushButton(Form)
        self.pushButton_2.setObjectName("pushButton_2")
        self.horizontalLayout_3.addWidget(self.pushButton_2)
        self.pushButton_4 = QtWidgets.QPushButton(Form)
        self.pushButton_4.setObjectName("pushButton_4")
        self.horizontalLayout_3.addWidget(self.pushButton_4)
        self.gridLayout.addLayout(self.horizontalLayout_3, 1, 2, 1, 1)
        self.textEdit = QtWidgets.QTextEdit(Form)
        self.textEdit.setObjectName("textEdit")
        self.gridLayout.addWidget(self.textEdit, 1, 3, 1, 1)

        self.retranslateUi(Form)
        QtCore.QMetaObject.connectSlotsByName(Form)

    def retranslateUi(self, Form):
        _translate = QtCore.QCoreApplication.translate
        Form.setWindowTitle(_translate("Form", "Form"))
        self.comboBox_21.setItemText(0, _translate("Form", "Hook Collection 1"))
        self.comboBox_21.setItemText(1, _translate("Form", "New Item"))
        self.pushButton_46.setText(_translate("Form", "No. of Hooks"))
        self.label_11.setText(_translate("Form", "Hook Collection Properties"))
        self.label_12.setText(_translate("Form", "Page: 1 | 2 | 3 ..."))
        self.pushButton_45.setText(_translate("Form", "Hook Collection"))
        self.pushButton_47.setText(_translate("Form", "Hook Collection Status"))
        self.comboBox_22.setItemText(0, _translate("Form", "Hook Collection 2"))
        self.comboBox_22.setItemText(1, _translate("Form", "New Item"))
        self.pushButton_48.setText(_translate("Form", "Hook Collection\n"
"Execution Status"))
        self.pushButton.setText(_translate("Form", "+Hook Collection"))
        self.pushButton_3.setText(_translate("Form", "Delete"))
        self.pushButton_2.setText(_translate("Form", "Edit"))
        self.pushButton_4.setText(_translate("Form", "Search"))
        self.textEdit.setHtml(_translate("Form", "<!DOCTYPE HTML PUBLIC \"-//W3C//DTD HTML 4.0//EN\" \"http://www.w3.org/TR/REC-html40/strict.dtd\">\n"
"<html><head><meta name=\"qrichtext\" content=\"1\" /><style type=\"text/css\">\n"
"p, li { white-space: pre-wrap; }\n"
"</style></head><body style=\" font-family:\'MS Shell Dlg 2\'; font-size:8pt; font-weight:400; font-style:normal;\">\n"
"<p style=\" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\"><span style=\" font-size:8.25pt; color:#a1a1a1;\">Name of Hook Collection</span></p></body></html>"))